test/io/wcm/testing/jenkins/pipeline/LibraryIntegrationTestBase.groovy

```diff
@@ -169,6 +149,11 @@ class LibraryIntegrationTestBase extends BasePipelineTest {
    */
   TeamsNotificationPluginMock teamsNotificationPluginMock
 
+  /**
+   * Mocks the Office365 connector plugin for MS Teams notifications
```

Comment:
this comment looks like a copy-paste error

Reply:
You are right, i will adjust that